stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/StreamAttachmentFactories.kt

```diff
@@ -64,18 +79,75 @@ public object StreamAttachmentFactories {
         giphySizingMode: GiphySizingMode = GiphySizingMode.ADAPTIVE,
         contentScale: ContentScale = ContentScale.Crop,
         skipEnrichUrl: Boolean = false,
+        onUploadContentItemClicked: (
+            Attachment,
+            List<AttachmentPreviewHandler>,
+        ) -> Unit = { attachment, previewHandlers ->
+            if (!attachment.isUploading()) {
+                previewHandlers
+                    .firstOrNull { it.canHandle(attachment) }
+                    ?.handleAttachmentPreview(attachment)
+            }
+        },
+        onLinkContentItemClicked: (context: Context, previewUrl: String) -> Unit = { context, previewUrl ->
+            context.startActivity(
+                Intent(
+                    Intent.ACTION_VIEW,
+                    Uri.parse(previewUrl)
+                )
+            )
+        },
+        onGiphyContentItemClick: (context: Context, Url: String) -> Unit = { context, url ->
+            context.startActivity(
+                Intent(
+                    Intent.ACTION_VIEW,
+                    Uri.parse(url)
+                )
+            )
+        },
+        onImageContentItemClicked: (
+            imagePreviewLauncher: ManagedActivityResultLauncher<ImagePreviewContract.Input, ImagePreviewResult?>,
+            message: Message,
+            attachmentPosition: Int,
+            skipEnrichUrl: Boolean,
+        ) -> Unit = { imagePreviewLauncher, messageClicked, clickedAttachmentPosition, skipEnrichUrl ->
```

Comment:
Don't think it's super important to use add `skipEnrichUrl` since it's available at the source however I passed it in for consistency.